app/src/gui/fields/maps/MapFormField.tsx

```diff
@@ -114,13 +133,12 @@ export function MapFormField({
     <div>
       <div>
         <MapWrapper
-          label={
-            props.label ? props.label : 'Get ' + props.featureType + ' from Map'
-          }
-          featureType={props.featureType}
+          label={label}
```

Comment:
This will always be the default label, not the supplied one.

Reply:
Fixed - good catch